app/src/main/java/nerd/tuxmobil/fahrplan/congress/sharing/LectureExport.kt

```diff
@@ -0,0 +1,42 @@
+package nerd.tuxmobil.fahrplan.congress.sharing
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import nerd.tuxmobil.fahrplan.congress.models.Lecture
+
+@JsonClass(generateAdapter = true)
+data class LectureExport(
```

Comment:
Is there any reason why you moved the files out of the `models` package again?

Reply:
I think some things got screwed up by my rebase. This was one of them, also the missing link in the readme.

Reply:
I thought so. Thank you for the cleanup in cddf05260159ac640e19044158964a2f6be86536.